crawler.c

```diff
@@ -645,13 +710,18 @@ enum crawler_result_type lru_crawler_crawl(char *slabs, const enum crawler_run_t
     uint32_t sid = 0;
     int starts = 0;
     uint8_t tocrawl[POWER_LARGEST];
+    bool hash_crawl = false;
 
     /* FIXME: I added this while debugging. Don't think it's needed? */
     memset(tocrawl, 0, sizeof(uint8_t) * POWER_LARGEST);
     if (strcmp(slabs, "all") == 0) {
         for (sid = 0; sid < POWER_LARGEST; sid++) {
             tocrawl[sid] = 1;
         }
+    } else if (strcmp(slabs, "hash") == 0) {
+        // TODO: Bounce if not metadump. expire eval is too closely tied to
```

Comment:
need to actually do this. making expire eval actually work will require more refactoring I don't really need to do right now; hash table walking for an expiration check is pointless.